internal/proxy/hook/authz/authz.go

```diff
@@ -30,12 +32,18 @@ type Authz struct {
 	// To skip all the next hooks and just respond back
 	escape hook.Service
 
+	Deps api.Deps
```

Comment:
Don't add deps dependency, use project service interface that you have defined instead

Reply:
Used Project Service to Get Organization.